syntax="proto3";

message Number{
    float number=1;
}

message NumberTwo{
    float number1=1;
    float number2=2;
}

service Calculator{
    rpc getSum(NumberTwo) returns (Number);
    rpc getProduct(NumberTwo) returns (Number);
    rpc getDiff(NumberTwo) returns (Number);
    rpc getDiv(NumberTwo) returns (Number);
    rpc getSqrt(Number) returns (Number);
}